tests/test_amity.py

```diff
@@ -1,146 +1,299 @@
-        self.assertTrue(os.path.isfile('test.db'))
-        os.remove('test.db')
+    def test_reallocate_person_to_fully_occupied_office_room(self):
+        """Test reallocating person to fully maximum number of occupants"""
+        self.amity.add_person("cn07", "Brian", "Rotich", "Fellow")
+        people_list = {"cn01": ["Sam", "wanjala", "Fellow"],
+                       "cn02": ["Gideon", "Gitau", "Fellow"],
+                       "cn03": ["Charles", "Muthini", "Fellow"],
+                       "cn04": ["Rogers", "Taracha", "Staff"],
+                       "cn05": ["Mahad", "Walusimbi", "Fellow"],
+                       "cn06": ["Percila", "Njira", "Staff"]}
+        self.amity.create_room("office", ["Mordor"])
+        for empno in people_list:
+            self.amity.add_person(empno, people_list[empno][0], people_list[empno][1], people_list[empno][2])
+
+        self.assertEqual(self.amity.reallocate_person("cn07", "Mordor"), "Sorry the Office is currently fully "
+                                                                         "occupied!")
+
+    def test_reallocate_person_to_fully_occupied_livingspace_room(self):
+        """Test reallocating person to fully maximum number of occupants"""
+        self.amity.add_person("cn05", "Brian", "Rotich", "Fellow", "Y")
+        people_list = {"cn01": ["Sam", "wanjala", "Fellow", "Y"],
+                       "cn02": ["Gideon", "Gitau", "Fellow", "Y"],
+                       "cn03": ["Charles", "Muthini", "Fellow", "Y"],
+                       "cn04": ["Rogers", "Taracha", "Fellow", "Y"]}
+        self.amity.create_room("lspace", ["Shell"])
+        for empno in people_list:
+            self.amity.add_person(empno, people_list[empno][0], people_list[empno][1], people_list[empno][2],
+                                  people_list[empno][3])
 
-    def test_reallocate_person_to_fully_occupied_room(self):
-        people_list = {"Sam": "Fellow", "Gideon": "Fellow", "Charles": "Fellow", "Rogers": "Staff", "Mahad": "Fellow",
-                       "Percila": "Staff"}
-        self.amity.create_room("Office", "Kenya")
-        for name in people_list:
-            self.amity.add_person(name, people_list[name])
-        self.amity.add_person("Batian", "Fellow")
-        self.assertEqual(self.amity.reallocate_person("Batian7", "Kenya"), "Sorry the office is fully occupied")
+        self.assertEqual(self.amity.reallocate_person("cn05", "Shell"), "Sorry the LivingSpace is currently fully occupied!")
 
     def test_reallocate_staff_to_living_space(self):
-        self.amity.create_room("Office", "Longonot")
-        self.amity.add_person("chelimo", "Staff")
-        self.amity.create_room("lspace", "Shell")
-        allocating_staff_living_space = self.amity.reallocate_person("chelimo", "Shell")
-        self.assertEqual(allocating_staff_living_space, "Sorry you cannot allocate staff living space")
+        """test reallocating staff to a living space room"""
```

Comment:
this is a test for living space, the office part seems unnecessary, lets have a discussion on this